esmvalcore/preprocessor/_io.py

```diff
@@ -274,3 +289,55 @@ def _write_ncl_metadata(output_dir, metadata):
     write_ncl_settings(info, filename)
 
     return filename
+
+
+def _concatenate_two_overlapping_cubes(cubes):
+    """Concatenate time-overlapping cubes (two cubes only)."""
+    # get time end points
+    time_1 = cubes[0].coord('time')
```

Comment:
maybe rename to `time_0`, `time_1`, etc. so it is in line with the indices.

Reply:
meh, it's cube1, cube2 etc :grin: